src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -1031,6 +1031,68 @@ void jit_is_nan_emitter::register_table_entries() {
     push_arg_entry_of("zero", 0x00000000, true);
 }
 
+/// LOGICAL_NOT ///
+jit_logicalnot_emitter::jit_logicalnot_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
```

Comment:
minor: We already have logical not emitter [implemented ](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_eltwise_emitters.cpp#L1426)for x64 architecture. I would ask you to aling naming with it: `jit_logicalnot_emitter` -> `jit_logical_not_emitter`